### PR TITLE
[TACHYON-795] Write more entries in FileSystemMaster

### PR DESCRIPTION
**Pull request by uronce-cc**
After this PR, journal for `FileSystemMaster` is done.

### CODE REVIEW

**Comment by uronce-cc** [comment]
@gpang PTAL

File: servers/src/main/java/tachyon/master/next/filesystem/FileSystemMaster.java [hunk 1]
```diff
@@ -388,14 +394,30 @@ public long getNewBlockIdForFile(long fileId) throws FileDoesNotExistException {
   public boolean deleteFile(long fileId, boolean recursive)
       throws TachyonException, FileDoesNotExistException {
     // TODO: metrics
+    long opTimeMs = System.currentTimeMillis();
+    boolean ret = deleteFileInternal(fileId, recursive, opTimeMs);
+    writeJournalEntry(new DeleteFileEntry(fileId, recursive, opTimeMs));
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/next/filesystem/FileSystemMaster.java]: The writing/flushing of the journal should be in the same critical section as the operation it is logging. So, that means it should be in the same synchronized block as the actual delete.

**Comment by gpang** [comment]
Thanks for the PR, @uronce-cc ! I just had one comment on how the journal should be written with respect to locks.

**Comment by uronce-cc** [comment]
@gpang done.

**Comment by gpang** [comment]
One of the lines are too long, which is failing checkstyle.

**Comment by gpang** [comment]
LGTM

**Pull request merged**